takoyaki/src/main/java/frc/robot/subsystems/CameraWithPID.java

```diff
@@ -0,0 +1,31 @@
+package frc.robot.subsystems;
+
+import frc.robot.subsystems.Camera;
+
+import org.photonvision.PhotonCamera;
+import org.photonvision.PhotonUtils;
+import org.photonvision.targeting.PhotonTrackedTarget;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.math.controller.PIDController;
+
+public class CameraWithPID {
+  private Camera camera;
+  private PIDController controller;
```

Comment:
nit: mark these fields as final
```suggestion
  private final Camera camera;
  private final PIDController controller;
```